frontend/src/components/TitleBar/Tags/ClassTagAdder.tsx

```diff
@@ -18,7 +18,7 @@ type SimpleCourse = {
   readonly noSpaceName: string;
 };
 
-type Props = { readonly fuse: Fuse<SimpleCourse> | null };
+type Props = { readonly fuse: Fuse<SimpleCourse, FuseOptions<SimpleCourse>> | null };
```

Comment:
This big change accidentally triggers some version bump. Fixed TS error as a result.